database/mssql/entrypoint.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
 # Start SQL Server
-/opt/mssql/bin/sqlservr & /usr/config/setup.sh
+echo "Start MS SQL Server and run scripts"
 
+if ! whoami &> /dev/null; then
```

Comment:
This is required because of the Microsoft bug where it cannot run `sqlcmd` in an OpenShift container.  OpenShift assigns a random UID user account to the pod, which Microsoft doesn't like.